Document the csv parser's `asObjects` option (#1850)
grafana/k6-docs: docs/sources/k6/next/javascript-api/k6-experimental/csv/Options.md

| Property      | Type              | Description                                                                                               |
| :------------ | :---------------- | :-------------------------------------------------------------------------------------------------------- |
| delimiter     | string            | The character used to separate fields in the CSV file. Default is `','`.                                  |

> | asObjects     | boolean           | Whether to parse the CSV file as an array of objects. Default is `false`.                                 |

Reviewer: ```suggestion
| asObjects     | boolean           | Whether to parse the CSV file's lines as JavaScript objects. If it is false then they are parsed as arrays. Default is `false`.                                 |
```

> | asObjects     | boolean           | Whether to parse the CSV file's lines as JavaScript objects. If it is false then they are parsed as arrays. Default is `false`.                                 |

Reviewer 2: ```suggestion
| asObjects     | boolean           | Whether to parse the CSV file's lines as JavaScript objects. If it is `false` then they are parsed as arrays. Default is `false`.                                 |
```

> | asObjects     | boolean           | Whether to parse the CSV file's lines as JavaScript objects. If it is `false` then they are parsed as arrays. Default is `false`.                                 |

Reviewer 3: ```suggestion
| asObjects     | boolean           | Whether to parse the CSV file's lines as JavaScript objects. If it is `false`, then they are parsed as arrays. Default is `false`.                                 |
```
